### PR TITLE
Fix function rewrite

### PR DESCRIPTION
**Pull request by pascalgrimaud**
When the needle is not found, the function `write` in `util.js` inserts the value at the beginning of the file, instead of displaying the error.

2 fix :
* init `otherwiseLineIndex` with -1, so it will throw error if the needle is not found
* it seems the `'text'.yellow` doesn't work any more, so the error was never displayed! I replace it by `chalk.yellow`

@atomfrede : as you work on module, can you confirm this issue ? just delete/change the needle in the `build.gradle` file and launch your module. Your dependency should be insert at the beginning of the file

And, if you and @jdubois can check my modifications too, because it's critical!
Thanks in advance

### CODE REVIEW

**Comment by atomfrede** [comment]
Yes, I can confirm the issue. Should be definitly fixed before the nest release.

**Comment by moifort** [comment]
:+1:

**Comment by jdubois** [comment]
No time to test, but yes I know the issue and this looks good, so I'm merging this as this can't be worse that we currently have.
I'll do some serious testing for all this at the beginning of next week, before the release.

File: script-base.js [hunk 1]
```diff
@@ -33,7 +34,7 @@ Generator.prototype.addJavaScriptToIndex = function (script) {
             ]
         });
     } catch (e) {
-        console.log('\nUnable to find '.yellow + fullPath + '. Reference to '.yellow + script + '.js ' + 'not added.\n'.yellow);
+        console.log(chalk.yellow('\nUnable to find ') + fullPath + chalk.yellow('. Reference to ') + script + '.js ' + chalk.yellow('not added.\n'));
```
Comment (moifort) [inline script-base.js]: Why don't put all the ligne in yellow like:    
``` 
console.log(chalk.yellow('\nUnable to find '+ fullPath + '. Reference to '+ script + '.js ' + 'not added.\n'));
```

**Pull request merged**